.mergify.yml

```diff
@@ -173,18 +176,19 @@ pull_request_rules:
       comment:
         message: Your hotfix is failing CI @{{author}} 🥺
 
-  - name: release notes update
+  - name: feature requirements
     conditions:
       - or:
           - "-title~=^feat"
           - and:
-              - "title~=^feat"
-              - files~=^releasenotes/notes
+            - files~=^releasenotes/notes
+            - files~=^doc/source
```

Comment:
do you want to add a label that can bypass this? just in case it's just an improvement with no doc change?
ditto for relnote?